File: 251004/Ryabchikov/tweet-service/src/main/java/by/ryabchikov/tweet_service/dto/tweet/TweetResponseTo.java
package by.ryabchikov.tweet_service.dto.tweet;

import lombok.Builder;

import java.time.LocalDateTime;


@Builder
public record TweetResponseTo(
        Long id,
        Long creatorId,
        String title,
        String content,
        LocalDateTime createdTime,
        LocalDateTime lastModifiedTime
) {
}

File: 251004/Kiseliov/lab2/TaskSQL/TaskSQL/Dto/Response/TweetResponseTo.cs
using System.Text.Json.Serialization;

namespace TaskSQL.Dto.Response;

public record TweetResponseTo(
    [property: JsonPropertyName("id")] long id,
    [property: JsonPropertyName("creatorId")]
    long creator_id,
    [property: JsonPropertyName("title")] string Title,
    [property: JsonPropertyName("content")]
    string Content
);